src/components/report-problem.style.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+export default {
+  camera: {
```

Comment:
can these just be abstracted into something like

`
container: {
  flex: 1
}
`

and just re-use `container` on all these components rather than re-defining the same layout rules over and over?